SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -1159,8 +1159,10 @@ void CFlowOutput::SetVolumeOutputFields_ScalarSolution(const CConfig* config){
   }
 
   if (config->GetKind_Species_Model() != SPECIES_MODEL::NONE) {
-    for (unsigned short iVar = 0; iVar < config->GetnSpecies(); iVar++)
+    for (unsigned short iVar = 0; iVar < config->GetnSpecies(); iVar++) {
       AddVolumeOutput("SPECIES_" + std::to_string(iVar), "Species_" + std::to_string(iVar), "SOLUTION", "Species_" + std::to_string(iVar) + " mass fraction");
+      AddVolumeOutput("DIFFUSIVITY_" + std::to_string(iVar), "Diffusivity_" + std::to_string(iVar), "PRIMITIVE", "Diffusivity_" + std::to_string(iVar));
```

Comment:
This function is for solution quantities not for primitives, if you do this here you may break the restarts.
Moving to SetVolumeOutputFields_ScalarResidual should be safer.

Reply:
Move to SetVolumeOutputFields_ScalarResidual. Thank you for your comment!!